databuilder/extractor/glue_extractor.py

```diff
@@ -0,0 +1,82 @@
+import boto3
```

Comment:
do you need to put boto3 into requirement file?

Reply:
That's the only thing I forgot :man_facepalming:

Reply:
Just added boto3 to the requirements